packages/components/src/components/buttons/buttonStyleUtils.ts

```diff
@@ -7,10 +7,9 @@ import { hexToRgba } from '@suite-common/suite-utils';
 const SUBTLE_ALPHA = 0.12;
 const SUBTLE_ALPHA_HOVER = 0.2;
 
-export type ButtonVariant = Extract<
-    UIVariant,
-    'primary' | 'secondary' | 'tertiary' | 'info' | 'warning' | 'destructive'
->;
+export type ButtonVariant =
+    | Extract<UIVariant, 'primary' | 'secondary' | 'tertiary' | 'info' | 'warning' | 'destructive'>
+    | 'welcome';
```

Comment:
Please do not add non-standard UI variants 🙏

Reply:
I knew you won't like it :) This button variant is really non-standard. The proposed style of the buttons/navigation items in the welcome screen isn't used anywhere else yet, unfortunately, even if it is similar to the style of nav item.

The design already exists for some time so I expected I could rely on it. Seemed that we really wanted a specific style for Welcome screen. I know this change here is kinda problematic also because we want to divide components into general ones and Suite specific ones. And this change violates this effort. However trying to use NavigationItem instead of button produced even more problems.. 

It seems to me some of our existing components lack more variety, properties, more functionality that prevents them to be nicely used for different screens and use cases without overwriting their styles or without using other not ideal ways of achieving the expected. And so the expected non-standard look is not easily achievable using our standard components.

Reply:
We should start another effort - to improve the existing components so they could be easily adaptable to what we want to display.